msctl

```diff
@@ -2483,7 +2504,16 @@ case "$1" in
       exit 1
     fi
   ;;
-  update)
+  update|force-update)
+    # Handle the force option.
+    if [ "$1" = "force-update" ]; then
+      if [ -s $VERSIONS_JSON ]; then
+        printf "Removing cached version manifest.\n"
+      fi
+      rm -f $VERSIONS_JSON
```

Comment:
Should we handle this the same way `updateVersionsJSON` does by renaming `version_manifest.json` to `version_manifest.json.bak`